Dockerfile

```diff
@@ -0,0 +1,92 @@
+FROM registry.access.redhat.com/ubi8:8.5
+
+ARG OPENRESTY_RPM_VERSION="1.19.3"
+
+LABEL summary="3scale's API gateway (APIcast) is an OpenResty application which consists of two parts: Nginx configuration and Lua files." \
+      description="APIcast is not a standalone API gateway therefore it needs connection to the 3scale API management platform. The container includes OpenResty and uses LuaRocks to install dependencies (rocks are installed in the application folder)." \
```

Comment:
```suggestion
      description="APIcast is not a standalone API gateway. It needs connection to the 3scale API management platform. The container includes OpenResty and uses LuaRocks to install dependencies. Install LuaRocks in the application folder." \
```

Reply:
this is not a user guide, it is a description of the image.